node/src/command.rs

```diff
@@ -95,6 +95,7 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
 			&include_bytes!("../res/chain_specs/litmus.json")[..],
 		)?),
 		// Rococo
+		"rococo-dev" => Box::new(chain_specs::rococo::get_chain_spec_dev()),
```

Comment:
What's the reason for adding rococo-dev here?

Reply:
> What's the reason for adding rococo-dev here?

for benchmarking, see https://github.com/litentry/litentry-parachain/issues/310